freeipa/src/main/resources/freeipa-salt/salt/monitoring/scripts/cipa.py

```diff
@@ -5,7 +5,12 @@
 import shutil
 import sys
 
-MAIN_PATCHED_SCRIPT="/usr/local/lib/python3.6/site-packages/checkipaconsistency/main.py"
+if sys.version_info.major == 3:
+    print("Python3." + str(sys.version_info.minor) + " is detected.")
+elif sys.version_info.major == 2:
+    print("Python2." + str(sys.version_info.minor) + " is detected.")
+MAIN_PATCHED_SCRIPT = "/usr/local/lib/python" + str(sys.version_info.major) + "." + str(sys.version_info.minor) + "/site-packages/checkipaconsistency/main.py"
+print("MAIN_PATCHED_SCRIPT_PATH: " + MAIN_PATCHED_SCRIPT)
```

Comment:
I can agree with this. :) Basically I was thinking on it but i did not know where is this used. Now I have changed it but I left last print. In case of problem for version check debugging it is also enough.

Reply:
I would remove the print from there too, and add it to line 30/31 as extra info where it exits if it's not able to find the binary